Makefile

```diff
@@ -66,12 +66,14 @@ coverage: ## check code coverage quickly with the default Python
 	$(BROWSER) htmlcov/index.html
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/mlbench.rst
-	rm -f docs/modules.rst
-	sphinx-apidoc -o docs/ mlbench
```

Comment:
I remove the three lines because the `master/`, `refimpls/`, `worker/` in `mlbench` are not python packages. So the `sphinx-apidoc` always generates an empty docs. I am not sure how we are going to arrange those three `master/`, `refimpls/`, `worker/` in the docs. 
Are we going to add them separately, or keep it as it is and add `__init__.py` instead?

Reply:
As mentioned in the other comment, you can just add `refimpls.rst` inside `modules.rst` toc tree to get it listed under modules. We can add the master node the same way.  

We will have to do a big refactoring anyways, since `worker/` should use the same metrics code framework as `refimpls` does right now, plus since `worker/` uses pytorch as well, it should be moved into `refimpls/pytorch/` in the future.

But for now we can just ignore `worker/` in the docs.